libs/designer-ui/src/lib/dictionary/index.tsx

```diff
@@ -64,8 +67,21 @@ export const DictionaryEditor: React.FC<DictionaryEditorProps> = ({
     onChange?.({ value: collapsedValue, viewModel: { items: isValid ? items : undefined } });
   };
 
+  const collapseIconDescription = intl.formatMessage({
+    defaultMessage: 'Switch to text mode',
+    description: 'Label for switching dictionary to text mode',
+  });
+
+  const expandIconDescription = intl.formatMessage({
+    defaultMessage: 'Switch to key value mode',
+    description: 'Label for switching dictionary to key value mode',
+  });
+
```

Comment:
I got the idea to put this here based on the code that is currently in EditorCollapseToggle -- currently we have the same toggle button text for array, authentication, and dictionary editor toggle buttons -- all of them currently have the array editor toggle text.

This code change fixes it for dictionary editor. I put it here in DictionaryEditor because dictionary editor should know what it wants its toggle button text to be

Reply:
@Eric-B-Wu to comment here.
@Ram-G - And you do not want this to be fixed for Authentication editor because??? Http action is still used in PA portal right?

Reply:
Removed this change from this PR. Will do in followup